lib/services/elasticsearch.js

```diff
@@ -38,6 +38,60 @@ function ElasticSearch (kuzzle, options, config) {
     }
   });
 
+  this.errorMessagesMapping = [
+    {
+      // [illegal_argument_exception] object mapping [titi] can't be changed from nested to non-nested
+      regex: /^\[illegal_argument_exception\] object mapping \[(.*?)\] can't be changed from nested to non-nested$/,
+      replacement: 'Can not change mapping for field "$1" from nested to another type'
+    },
+    {
+      // [illegal_argument_exception] object mapping [baz] can't be changed from non-nested to nested
+      regex: /^\[illegal_argument_exception\] object mapping \[(.*?)\] can\'t be changed from non-nested to nested$/,
+      replacement: 'Can not change mapping for field "$1" from object to another type'
```

Comment:
Same remark: "another type" is quite vague.
Why not `replacement: 'Can not change mapping for field "$1" from a non-nested to a nested object'` ?